vtysh/vtysh_config.c

```diff
@@ -104,22 +106,22 @@ struct configuration {
 	struct config_master_hash_head hash_master;
 };
 
-static struct config *config_get(int index, const char *line)
+static struct config *__config_get(vector vec, int index, const char *line)
```

Comment:
Not sure if this is a good practice or not as per the frr coding guidelines to start the function name with '__'. Suggest to rename this to something without '__' at the beginning.

Reply:
Maybe rename this to 'config_gett_from_vec()'. It seems to be what its really doing.